### PR TITLE
linode create

### PR DESCRIPTION
**Pull request by mjhea0**

### CODE REVIEW

**Comment by mjhea0** [comment]
any idea how to see how many current linodes and account has?

**Comment by madflojo** [comment]
https://www.linode.com/api/linode/linode.list

I would say count the number of list items.

**Comment by madflojo** [comment]
BTW I just created a bounty for monitoring linode node status. Which would need the same end point, figured I would let you know since you're working on this one.

**Comment by mjhea0** [comment]
test!

**Comment by madflojo** [comment]
So I noticed two things while testing this.

1. The reaction always returns false even when successful
2. The limit doesn't actually limit it only determines if the reaction returns True or False but nodes are still created

**Comment by mjhea0** [comment]
Yeah, I saw that too. I am having a hard time testing a lot of this. Are
you just manually testing on your end?

**Comment by madflojo** [comment]
Yeah, 

Basically just launch a version with fig and then create a monitor/reaction.

**Comment by mjhea0** [comment]
ready to test again

**Comment by mjhea0** [comment]
updated

**Comment by mjhea0** [comment]
@madflojo were you able to test again?

**Comment by madflojo** [comment]
Hey sorry man, have been working to finish up a chapter so haven't had a chance. Will probably be working on it all day so I'll let you know when I've tested it again.

**Comment by mjhea0** [comment]
No worries. Super busy on my end as well. @madflojo

File: src/actions/actions/linode-create/__init__.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                logger.info(line)
+                return False
+            else:
+                line = 'linode-create: Request to {0} sent for monitor {1} - \
+                    Successful'.format(url, jdata['cid'])
+                logger.info(line)
+            return True
+    else:
+        line = 'linode-create: Request to {0} sent for monitor {1} - \
+            Skipped (upper_limit >= current linodes)'.format(url, jdata['cid'])
```
Comment (madflojo) [inline src/actions/actions/linode-create/__init__.py]: actioner_1      | 2015-01-15 22:46:04,049 - actions.actioner - ERROR - Got error when attempting to run reaction linode-create for monitor a4111012-e956-492a-a219-f3d4e82ec4b4
actioner_1      | 2015-01-15 22:46:04,050 - actions.actioner - ERROR - local variable 'url' referenced before assignment

**Comment by madflojo** [comment]
Ok tested it out and after a few modifications I got it working 100%. 

FYI the modifications were.

1. Moved url variable above the if statement
2. Flipped the < to a > on the limit vs current status. As the logic should be if the upper limit is greater than the current number than you are safe to continue.
3. Increased the timer as I was getting request timeouts from linode
4. Revised logging for scenario number 2

I'm going to bust out some of the HTML stuff in this PR and then merge it manually too btw.

**Pull request merged**